third_party/github.com/hashicorp/go-retryablehttp/client.go

```diff
@@ -601,7 +602,7 @@ func (c *Client) Do(req *Request) (*http.Response, error) {
 			if c, ok := body.(io.ReadCloser); ok {
 				req.Body = c
 			} else {
-				req.Body = io.NopCloser(body)
+				req.Body = ioutil.NopCloser(body)
```

Comment:
An update to go1.19 shouldn't change this. Any idea why this happened? Also, why do we need go code here?

Reply:
(the go code is due to the license terms of the particular library I think)